tests/phpunit/FilenameTestCase.php

```diff
@@ -73,7 +75,7 @@ public function doesNotWorkWithInvalidName(): void
         expect(function (): void {
             new RecursiveDirectoryIterator($this->rootDir . '/lost found/');
         })->throws(UnexpectedValueException::class)
-          ->message(contains('failed to open dir'));
+          ->message(contains('ailed to open dir'));
```

Comment:
```suggestion
          ->message(contains('failed to open dir'));
```

Reply:
I think the change to remove the leading f is correct, AFAIR PHP 8 changed the error message from „failed ...“ to „Failed ...“. As `contains()` is case sensitive and at the moment there‘s no case insensitive variant for it it‘s probably the best to just remove the f. However, a comment might be nice to have so that confusion doesn‘t rise again at a later point in time.

Reply:
That was indeed the reason for this change, I will add a comment and reorder the commits a bit to make this a bit less messy